### PR TITLE
typstfmt 0.2.8 (new formula)

### PR DESCRIPTION
**Pull request by pavelzw**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/t/typstfmt.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  homepage "https://github.com/astrale-sharp/typstfmt"
+  url "https://github.com/astrale-sharp/typstfmt/archive/refs/tags/0.2.7.tar.gz"
+  sha256 "17db7ddb83a8e2e43b3b34f19dcc8cbd3585aea34e05a5689e6f5beda27ac523"
+  license "MIT"
+  head "https://github.com/astrale-sharp/typstfmt.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: ".")
```
Comment (chenrui333) [inline Formula/t/typstfmt.rb:12]:
```suggestion
    system "cargo", "install", *std_cargo_args
```

File: Formula/t/typstfmt.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Typstfmt < Formula
+  desc "Formatter for typst"
+  homepage "https://github.com/astrale-sharp/typstfmt"
+  url "https://github.com/astrale-sharp/typstfmt/archive/refs/tags/0.2.7.tar.gz"
+  sha256 "17db7ddb83a8e2e43b3b34f19dcc8cbd3585aea34e05a5689e6f5beda27ac523"
+  license "MIT"
```
Comment (chenrui333) [inline Formula/t/typstfmt.rb:6]: - there is some license issue, https://github.com/astrale-sharp/typstfmt/issues/153

Comment (pavelzw) [reply Formula/t/typstfmt.rb:6]: Should we change it to `MIT OR Apache-2.0` here?

Comment (SMillerDev) [reply Formula/t/typstfmt.rb:6]: We should be sure what the license is before doing either.

Comment (pavelzw) [reply Formula/t/typstfmt.rb]: updated it to `one_of: ["MIT", "Apache-2.0"]`

**Comment by chenrui333** [comment]
need a new release to include the Apache LICENSE in the source tarball.

**Comment by chenrui333** [comment]
ping upstream again for the license change release.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@pavelzw, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**